toolboxTreppe/toolboxTreppe.ino

```diff
@@ -103,10 +103,11 @@ void animation1(uint8_t& y, uint8_t& x, uint8_t& gegenstrecke) {
 }
 //----------------------------- Debug Funktion ----------------
 
-void debugPrint(const char* output[], ...) {
+void debugPrint(const char* output, ...) {
   va_list args;
-  if (digitalRead(buttonPin1) && digitalRead(buttonPin2)) {
-    Serial.println(output, args);
+  char* buffer;
```

Comment:
has to be a allocated buffer:
char buffer[1024]; //size of 1024 is not too small I hope

Reply:
true will change that and yes I guess nobody should print more than 1024 chars per line when he wants a readable debug output